### PR TITLE
docs: misc

### PR DESCRIPTION
**Pull request by bwanglzu**
see it lively [here](https://ft-docs-misc--jina-docs.netlify.app/docs-misc/)

1. use `CLIPLoss` in get started section for CLIP documentation
2. add auto-doc generated developer reference back
3. add Jina echosystem section
4. documentation style fix.
5. add back the documentation release CD, not tested (it will replace the current OSS finetuner docs after release).

F1B473E2-9351-4F0C-A11B-D694A1262E36

### CODE REVIEW

**Comment by bwanglzu** [comment]
wired, let me see

**Review by gmastrapas — COMMENTED** [review]
Jina NOW and jcloud are missing from the ecosystem

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Support passing callbacks to the run configuration. ([#415](https://github.com/jina-ai/finetuner/pull/415))
 
+- Docs add developer reference, Jina ecosystem and style fix. ([#423](https://github.com/jina-ai/finetuner/pull/423))
+
```
Comment (jupyterjazz) [inline CHANGELOG.md:22]: Let's put this below #416 in the changelog

**Review by jupyterjazz — COMMENTED** [review]
Is it possible to remove some of the modules from the developer reference? Showing `constants` and `names` modules is just unnecessary.

File: .github/workflows/force-docs-build.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          python-version: 3.7
+      - name: Build doc and push to gh-pages
+        run: |
+          git config --local user.email "dev-bot@jina.ai"
+          git config --local user.name "Jina Dev Bot"
+          pip install .
+          mkdir gen-html
+          cd docs
+          pip install -r requirements.txt
+          pip install --pre -U furo
```
Comment (guenthermi) [inline .github/workflows/force-docs-build.yml:41]: why do you install furo separately? Shouldn't it be specifified in the requirements.txt you installed just before.

Comment (bwanglzu) [reply .github/workflows/force-docs-build.yml:41]: not sure, copied from main branch, this is not tested, will clear it up when we merge finetuner client to main branch

**Review by jupyterjazz — APPROVED** [review]
:rocket:

**Review by jemmyshin — APPROVED** [review]
LGTM!

**Review by gmastrapas — APPROVED** [review]
Looks good 🚀

**Pull request merged**